### PR TITLE
100 Theorems

### PR DESCRIPTION
**Pull request by fredrik-bakke**
Adds a page about our progress on [_Formalizing 100 Theorems_](https://www.cs.ru.nl/~freek/100/).

### CODE REVIEW

File: src/elementary-number-theory/positive-integers.lagda.md [hunk 1]
```diff
@@ -122,6 +126,9 @@ is-nonzero-is-positive-ℤ {inr (inr x)} H ()
 is-set-positive-ℤ : is-set positive-ℤ
 is-set-positive-ℤ =
   is-set-type-subtype subtype-positive-ℤ is-set-ℤ
+
+set-positive-ℤ : Set lzero
+set-positive-ℤ = positive-ℤ , is-set-positive-ℤ
```
Comment (EgbertRijke) [inline src/elementary-number-theory/positive-integers.lagda.md:131]:
```suggestion
positive-ℤ-Set : Set lzero
positive-ℤ-Set = positive-ℤ , is-set-positive-ℤ
```

**Review by EgbertRijke — APPROVED** [review]
Very nice addition to the library! It's great to see a record of the theorems from Freek's list that we already have in the library.

**Comment by EgbertRijke** [comment]
This PR is already on auto-merge, so I don't want to make any suggestions for changes in this PR. But would it be nice to add credit to the original formalizers of those theorems, and a GitHub tag for pull requests that complete a new theorem?

**Comment by fredrik-bakke** [comment]
Oh, yeah, that would be good!

**Comment by EgbertRijke** [comment]
Maybe we can do that with the OEIS sequences as well, although some of those sequences are quite silly.

**Pull request merged**